randomizer/Lists/CBLocations/JungleJapesCBLocations.py

```diff
@@ -224,7 +224,7 @@ def setSpawnPoint(self, points=[]):
         name="Up the slope to painting room",
         konglist=[Kongs.lanky, Kongs.tiny],
         region=Regions.JungleJapesMain,
-        logic=lambda l: (l.handstand and l.lanky) or (l.twirl and l.tiny),
+        logic=lambda l: (l.handstand and l.lanky) or (l.twirl and l.istiny),
```

Comment:
Shouldn't this also be `islanky`?

Reply:
Yeah it should, and it will! Good catch!